packages/dbml-parse/vite.config.ts

```diff
@@ -6,9 +6,7 @@ import dts from 'vite-plugin-dts';
 
 export default defineConfig({
   plugins: [
-    dts({
-      insertTypesEntry: true,
-    }),
+    dts(),
```

Comment:
Do we still need the [types option](https://github.com/holistics/dbml/blob/4a4f1a954f551b345fc5f33bdd91af9826a99712/packages/dbml-parse/package.json#L15) in package.json file after removing the `insertTypesEntry: true` option?

Reply:
Yeah anh :v that `insertTypeEntry` is redundant for `dts` anh, I remove that and it still generates `index.d.ts`, so `types` in `package.json` still points to that